cql3/statements/create_aggregate_statement.cc

```diff
@@ -36,10 +36,19 @@ seastar::future<shared_ptr<db::functions::function>> create_aggregate_statement:
     acc_types.insert(acc_types.end(), _arg_types.begin(), _arg_types.end());
     auto state_func = dynamic_pointer_cast<functions::scalar_function>(functions::functions::find(functions::function_name{_name.keyspace, _sfunc}, acc_types));
     if (!state_func) {
-        throw exceptions::invalid_request_exception(format("State function not found: {}", _sfunc));
+        sstring args;
+        for (int i = 0; i < acc_types.size(); i++) {
```

Comment:
Please consider using fmt::join instead of this loop. I think it makes the code nicer and shorter. E.g., you should be able to just print `fmt::join(acc_types, ", ")` and it will just do what you wanted.